util/patch/patch_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Accidental date change here?

Reply:
Do we need to change it back? I'm not opposed, mostly wondering in general what's the rule given the large rewrite

Reply:
I'm not sure that I've seen a place where we've bumped the copyright on an existing file. I'm also not finding any guidance on handling the year in the upstream community docs :man_shrugging:

Reply:
Yeah I don't think you ever change the year on existing files.